.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "compact-tools"]
+	path = compact-tools
+	url = https://github.com/OpenZeppelin/compact-tools
```

Comment:
I think adding this submodule adds unnecessary complexity and should be avoided. We should use a reference to a local copy of the compact tools for development until we're able to release it publicly, after which we can simply consume it as an npm package. Unfortunately, this means external contributors will not be able to build the project locally until compact-tools is released so we should prioritize delivering it ASAP

Reply:
We will need to change the setup action to checkout multiple repos like so https://github.com/actions/checkout#Checkout-multiple-repos-private.